lib/remark-lint-thai.js

```diff
@@ -55,6 +55,13 @@ const rules = [
       if (match[1] !== 'ๆ') return 'Add a space after ๆ'
     },
   },
+  // All about 'กัน' -> กรร
+  {
+    pattern: /กัน(ไกร|เชียง|โชก|ไตร)/g,
+    check(match) {
+      return `“${match[0]}” → “กรร${match[2]}”`
```

Comment:
Testing it on https://deploy-preview-3--remark-lint-thai.netlify.com and…

![image](https://user-images.githubusercontent.com/193136/63219767-4ac89480-c1a3-11e9-96ac-a6ab193abf41.png)

I think this should be `match[1]`...

Reply:
Oops! my bad. 😛